setup.py

```diff
@@ -1,103 +1,144 @@
+            pass
+
+        self.status("Building Source and Wheel (universal) distribution…")
+        os.system("{0} setup.py sdist bdist_wheel --universal".format(sys.executable))
+
+        self.status("Uploading the package to PyPI via Twine…")
+        os.system("twine upload dist/*")
+
+        self.status("Pushing git tags…")
+        os.system("git tag v{0}".format(about["__version__"]))
+        os.system("git push --tags")
+
+        sys.exit()
 
-version_namespace = {}
-for line in open(os.path.join(PROJECT_PATH, "sbi", "__version__.py")):
-    if line.startswith("__version__ = "):
-        exec(line, version_namespace)
 
 setup(
-    name="sbi",
-    version=version_namespace["__version__"],
-    description="Simulation-based inference.",
-    long_description=desc,
+    name=NAME,
+    version=about["__version__"],
+    description=DESCRIPTION,
+    long_description=long_description,
     long_description_content_type="text/markdown",
-    keywords="bayesian parameter inference system_identification simulator PyTorch",
-    url="http://mackelab.org/sbi",
-    author=(
-        "Álvaro Tejero-Cantero, Jakob H. Macke, Jan-Matthis Lückmann,"
-        " Conor M. Durkan, Michael Deistler, Jan Bölts."
-    ),
-    author_email="sbi@mackelab.org",
-    packages=find_packages(exclude=["tests"]),
+    author=AUTHOR,
+    author_email=EMAIL,
+    python_requires=REQUIRES_PYTHON,
+    url=URL,
+    packages=find_packages(exclude=["tests", "*.tests", "*.tests.*", "tests.*"]),
+    install_requires=REQUIRED,
+    extras_require=EXTRAS,
+    include_package_data=True,
     license="AGPLv3",
-    install_requires=[
-        "ipywidgets",
-        "joblib",
-        "jupyterlab",
-        "matplotlib",
-        "nbstripout",
-        "notebook",
-        "numpy",
-        "pillow",
-        "pyro-ppl",
-        "pyknos==0.11",
-        "scipy",
-        "tensorboard",
-        "torch>=1.4.0, !=1.5.0",  # See issue #37703 in PyTorch 1.5.0.
-        "tqdm",
-    ],
-    extras_require={
-        "dev": [
-            "autoflake",
-            "black",
-            "deepdiff",
-            "flake8",
-            "isort",
-            "mkdocs",
-            "mkdocs-material",
-            "markdown-include",
-            "mkdocs-redirects",
-            "mkdocstrings",
-            "nbconvert",
-            "pep517",
-            "pytest",
-            "pyyaml",
-            "scikit-learn",
-            "torchtestcase",
-            "twine",
-        ]
-    },
-    dependency_links=[],
     classifiers=[
+        # Full list: https://pypi.python.org/pypi?%3Aaction=list_classifiers
         "Development Status :: 3 - Alpha",
+        "License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)",
         "Intended Audience :: Developers",
         "Intended Audience :: Education",
-        "Intended Audience :: Information Technology",
         "Intended Audience :: Science/Research",
-        "Topic :: Adaptive Technologies",
```

Comment:
Why removing these classifiers?

Reply:
I took inspiration from the classifiers that `pyro` uses and adapted ours. They don't use `Topic ::` classifiers and I wasn't to sure about `Topic :: Games/Entertainment` and `Topic :: Adaptive Technologies` so I dropped topic classifiers altogether. Can put all or some back in, if you prefer!

Reply:
Yes, games/entertainment was on the edge but it contained 'simulation' which I thought may provide some use-cases (you never know).
```
        "Topic :: Adaptive Technologies",
        "Topic :: Scientific/Engineering",
        "Topic :: Scientific/Engineering :: Artificial Intelligence",
        "Topic :: Scientific/Engineering :: Mathematics",
        "Topic :: Games/Entertainment :: Simulation",
```
From these I'd say except the games maybe, all are relevant and could be used in searches.